tenant/service_bucket.go

```diff
@@ -157,8 +151,19 @@ func (s *BucketSvc) FindBuckets(ctx context.Context, filter influxdb.BucketFilte
 			Description:     "System bucket for task logs",
 		}
 
-		buckets = append(buckets, tb)
+		// if request is for first page, then add legacy system bucket to beginning of response and
+		// adjust offset and limit
+		// because this is the first bucket in the list, only return it if there is no After field
+		// if request is for non-first page then decrement offset based on number of legacy buckets
+		if o.Offset < 2 && o.After == nil {
```

Comment:
can you create a variable named in a way that describes what `2` means and use that, rather than the [magic number 2](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants) here?

Reply:
Great point, thank you!